build.gradle

```diff
@@ -787,6 +800,12 @@ subprojects {
     skipProjects = [ ":jmh-benchmarks", ":trogdor" ]
     skipConfigurations = [ "zinc" ]
   }
+
+  afterEvaluate {
```

Comment:
Maybe we can set spotless directly. For example:
```
  if (project.name in spotlessApplyModules) {
    apply plugin: 'com.diffplug.spotless'
    spotless {
      java {
        importOrder('kafka', 'org.apache.kafka', 'com', 'net', 'org', 'java', 'javax', '', '\\#')
        removeUnusedImports()
      }
    }
  }
```

Reply:
I change the PR,please review it @chia7712